### ISSUE TITLE
is there a way to store metadata in the model?

### ISSUE THREAD

**Issue by ophiry** [fchollet/keras#6404]
is it possible to add custom meta data to keras models that will be saved and loaded automatically?

the main need is to save the input preprocessing parameters, the train and test set used, etc.

**Comment by diggy** [comment]
I've spotted setting and getting of model metadata in this repo: https://github.com/nadbordrozd/neural_playground/blob/master/text_generation.py#L167

**Comment by ophiry** [comment]
interesting, but this field doesn't exist in keras models
